### ISSUE TITLE
Need to find if a variable or method can be private through lint

### ISSUE THREAD

**Issue by SAGARSURI** [passsy/dart-lint#12]
I am not able to find a lint rule which can analyse and tell me if the variable or method can be private.

**Comment by kuhnroyal** [comment]
I don't think that can be linted. This would need a much deeper analysis.

**Comment by rrifafauzikomara** [comment]
Yeah I need this too, looks like Android native can find a variable or method that can be private

**Comment by kvenn** [comment]
I'd also like this. Is this referring to variable that is currently public, but is not accessed outside the current file?
